### PR TITLE
Feature: get cover fees

### PR DESCRIPTION
**Pull request by savinnpm**

### CODE REVIEW

File: src/components/UI/organisms/PolicyFeesAndExpiry/PolicyFeesAndExpiry.jsx [hunk 1]
```diff
@@ -1,8 +1,11 @@
-export const PolicyFeesAndExpiry = ({ fees, daiValue, claimEnd }) => {
+import { monthNames } from "@/lib/dates";
+
+export const PolicyFeesAndExpiry = ({ fees, feeAmount, claimEnd }) => {
   const formatDate = () => {
     let date = new Date();
-    let dateToShow = claimEnd.substr(0, 3);
-    dateToShow = dateToShow + ` 31, ${date.getUTCFullYear()} 12:00 UTC`;
+    console.log(monthNames[(date.getMonth() + claimEnd - 1) % 12]);
```
Comment (flashburst) [inline src/components/UI/organisms/PolicyFeesAndExpiry/PolicyFeesAndExpiry.jsx:6]: Remove console.log

File: src/components/UI/organisms/cover-form/index.jsx [hunk 2]
```diff
@@ -90,6 +90,40 @@ export const CoverForm = ({
...
+        signerOrProvider
+      );
+      const { fee, rate } = response.result;
+      setFees(
+        parseFloat(convertFromUnits(rate).multipliedBy(100).toString()).toFixed(
+          2
+        )
+      );
+      setFeeAmount(parseFloat(convertFromUnits(fee).toString()).toFixed(3));
+      console.info("--------------------------------------");
```
Comment (flashburst) [inline src/components/UI/organisms/cover-form/index.jsx:117]: Remove console.info (s)

**Review by flashburst — CHANGES_REQUESTED** [review]

File: src/components/UI/organisms/PolicyFeesAndExpiry/PolicyFeesAndExpiry.jsx (hunk 1 above)
Comment (savinnpm) [reply src/components/UI/organisms/PolicyFeesAndExpiry/PolicyFeesAndExpiry.jsx]: done

File: src/components/UI/organisms/cover-form/index.jsx (hunk 2 above)
Comment (savinnpm) [reply src/components/UI/organisms/cover-form/index.jsx]: completed

**Review by flashburst — APPROVED** [review]

**Pull request merged**